src/poliastro/twobody/propagation.py

```diff
@@ -222,3 +478,36 @@ def propagate(orbit, time_of_flight, *, method=mean_motion, rtol=1e-10, **kwargs
     )
 
     return cartesian
+
+
+PROPAGATORS_SET = [
```

Comment:
Instead of building `PROPAGATORS_SET` manually, why don't we intersect 

```
ALL_PROPAGATORS = list(set(ELLIPTIC_PROPAGATORS) & set(...) & set(...))
```

?